tests/waku_rln_relay/rln/test_wrappers.nim

```diff
@@ -0,0 +1,131 @@
+import testutils/unittests
+
+import
+  ../../../waku/[waku_rln_relay, waku_rln_relay/rln, waku_rln_relay/rln/wrappers],
+  ./waku_rln_relay_utils,
+  ../../testlib/[simple_mock]
+
+import std/[options]
+import
+  chronicles,
+  chronos,
+  stew/shims/net as stewNet,
+  eth/keys,
+  bearssl,
+  stew/[results],
+  metrics,
+  metrics/chronos_httpserver
```

Comment:
Here we can reorder the imports so we only have 2 import statements instead of 4

```suggestion
import
  std/[options],
  testutils/unittests,
  chronicles,
  chronos,
  stew/shims/net as stewNet,
  eth/keys,
  bearssl,
  stew/[results],
  metrics,
  metrics/chronos_httpserver
  
  import
  ../../../waku/[waku_rln_relay, waku_rln_relay/rln, waku_rln_relay/rln/wrappers],
  ./waku_rln_relay_utils,
  ../../testlib/[simple_mock]
```

Reply:
Done